dspace-spring-rest/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -514,4 +519,420 @@ public void deleteViolatingConstraints() throws Exception {
                 .andExpect(status().isOk());
     }
 
+    @Test
+    public void patchByForbiddenUser() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        EPerson ePerson = EPersonBuilder.createEPerson(context)
+                                        .withNameInMetadata("John", "Doe")
+                                        .withEmail("Johndoe@fake-email.com")
+                                        .build();
+
+        List<Operation> ops = new ArrayList<Operation>();
+        ReplaceOperation replaceOperation = new ReplaceOperation("/netid", "newNetId");
+        ops.add(replaceOperation);
+        String patchBody = getPatchContent(ops);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        // should be forbidden.
+        getClient(token).perform(patch("/api/eperson/epersons/" + ePerson.getID())
+                .content(patchBody)
+                .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                   .andExpect(status().isForbidden());
```

Comment:
it will be better to check that no changes have been applied

Reply:
I agree with @abollini here. We should also add a check to see that no changes were made to the object... checking the return status is good, but we also want to ensure that the patch wasn't accidentally applied.  

Same with the other comment just below this one.

Reply:
I think we are good for the missing tests to ensure that the patch was not applied. I added them (they don't appear here in the diff summary.)

This one checks the netid to be sure it wasn't updated.    
  ```
  getClient(token).perform(patch("/api/eperson/epersons/" + ePerson.getID())
                .content(patchBody)
                .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
                   .andExpect(status().isForbidden());


        token = getAuthToken(admin.getEmail(), password);


        getClient(token).perform(get("/api/eperson/epersons/" + ePerson.getID()))
                        .andExpect(status().isOk())
                        .andExpect(jsonPath("$.netid", Matchers.is(nullValue())));
```